frameworks/Crystal/crystal/server.cr

```diff
@@ -14,9 +22,72 @@ server = HTTP::Server.new("0.0.0.0", 8080) do |context|
     response.status_code = 200
     response.headers["Content-Type"] = "text/plain"
     response.print "Hello, World!"
+  when "/db"
+    response.status_code = 200
+    response.headers["Content-Type"] = "application/json"
+    random_world.to_json(response)
+  when "/queries"
+    response.status_code = 200
+    response.headers["Content-Type"] = "application/json"
+
+    results = (1..sanitized_query_count(request)).map do
```

Comment:
You could use `JSON.build` to stream the array here without allocating, which would be faster.